src/REISE.jl

```diff
@@ -76,11 +76,11 @@ function run_scenario(;
     println("All preparation complete!")
     # While redirecting stdout and stderr...
     println("Redirecting outputs, see stdout.log & stderr.err in outputfolder")
-    redirect_stdout_stderr(stdout_filepath, stderr_filepath) do
+    #redirect_stdout_stderr(stdout_filepath, stderr_filepath) do
         # Loop through intervals
-        m = interval_loop(optimizer_factory, model_kwargs, solver_kwargs, interval,
+    m = interval_loop(optimizer_factory, model_kwargs, solver_kwargs, interval,
                           n_interval, start_index, inputfolder, outputfolder)
-    end
+    #end
```

Comment:
If the code is working, should we still be exiting the stdout redirection here?

Reply:
> If the code is working, should we still be exiting the stdout redirection here?

Yes. I wasn't planning to put this version on PR before the meeting, so I left a lot of debug lines in. I will remove them in a later commit.